src/ui/public/chrome/directives/global_nav/app_switcher/app_switcher.html

```diff
@@ -1,12 +1,11 @@
-
 <global-nav-link
   ng-repeat="link in switcher.links"
   ng-if="!link.hidden"
   is-active="link.active"
   is-disabled="link.disabled"
   tooltip-content="switcher.getTooltip(link)"
   on-click="switcher.ensureNavigation($event, link)"
-  href="link.active ? link.url : (link.lastSubUrl || link.url)"
+  url="link.active ? link.url : (link.lastSubUrl || link.url)"
   icon="link.icon"
-  title="link.title"
+  label="link.title"
```

Comment:
Angular applies the `href` and `title` attributes to the HTML element so we have to give them names that don't collide with the HTML spec.